activesupport/lib/active_support/core_ext/time/zones.rb

```diff
@@ -82,7 +82,8 @@ def use_zone(time_zone)
     def find_zone!(time_zone)
       return time_zone unless time_zone
 
-      ActiveSupport::TimeZone[time_zone] || raise(ArgumentError, "Invalid Timezone: #{time_zone}")
+      tz_reference = 'https://github.com/rails/rails/blob/main/activesupport/lib/active_support/values/time_zone.rb'
+      ActiveSupport::TimeZone[time_zone] || raise(ArgumentError, "Invalid Timezone: #{time_zone}! Find the valid identifiers in #{tz_reference}")
```

Comment:
I'm not sure about linking to github - the link could change over time, or the values could differ between versions. How about this as an alternative?

```suggestion
      ActiveSupport::TimeZone[time_zone] || raise(ArgumentError, "Invalid Timezone: #{time_zone}! Find the valid identifiers in #{ActiveSupport::TimeZone::MAPPING}")
```

Reply:
Thanks for this suggestion :-) I've amended it offline and added you as co-author of the commit. Is that OK?